english/buy-gameboy/buy-gameboy.md

```diff
@@ -0,0 +1,2 @@
+you realize that Pokemon is great and can still help you with your troubles, you find a man at a flea market selling Pokemon Red,
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.